src/components/charts/ServicesChart.tsx

```diff
@@ -7,17 +7,17 @@ export default function ServicesChart() {
 
   const serviceCategories = getSubcategories('services sociétaux')
```

Comment:
Maybe you should return empty array from `getSubcategories` to avoid this diff. See comment below `getSubcategories` already returns `[]`